rct229/rulesets/ashrae9012019/section19/section19rule13.py

```diff
@@ -0,0 +1,261 @@
+    HVAC_SYS,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_baseline_system_types import (
+    get_baseline_system_types,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_dict_of_zones_and_terminal_units_served_by_hvac_sys import (
+    get_dict_of_zones_and_terminal_units_served_by_hvac_sys,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_fan_system_object_supply_return_exhaust_relief_total_power_flow import (
+    get_fan_system_object_supply_return_exhaust_relief_total_power_flow,
+)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO, CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_1,
+    HVAC_SYS.SYS_2,
+    HVAC_SYS.SYS_3,
+    HVAC_SYS.SYS_4,
+    HVAC_SYS.SYS_5,
+    HVAC_SYS.SYS_6,
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_8,
+    HVAC_SYS.SYS_11,
+    HVAC_SYS.SYS_12,
+    HVAC_SYS.SYS_13,
+]
+
+LIGHTING_SPACE = schema_enums["LightingSpaceOptions2019ASHRAE901TG37"]
+TEMP_DELTA_17F = 17.0 * ureg("R")
+TEMP_DELTA_20F = 20.0 * ureg("R")
+
+
+class Section19Rule13(RuleDefinitionListIndexedBase):
+    """Rule 13 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule13, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            each_rule=Section19Rule13.HVACRule(),
+            index_rmr="baseline",
+            id="19-13",
+            description="For baseline system types 1-8 and 11-13, system design supply airflow rates shall be based on a supply-air-to-room temperature set-point difference of 20°F or the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater. For systems with multiple zone thermostat setpoints, use the design set point that will result in the lowest supply air cooling set point or highest supply air heating set point.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.8.1 and Exception 1",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_instances/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def create_data(self, context, data):
+        rmi_b = context.baseline
+        rmi_p = context.proposed
+
+        dict_of_zones_and_terminal_units_served_by_hvac_sys_b = (
+            get_dict_of_zones_and_terminal_units_served_by_hvac_sys(rmi_b)
+        )
+
+        zone_info = {}
+        for hvac_id_b in dict_of_zones_and_terminal_units_served_by_hvac_sys_b:
+            for zone_id_b in dict_of_zones_and_terminal_units_served_by_hvac_sys_b[
+                hvac_id_b
+            ]["zone_list"]:
+                zone_info[zone_id_b] = {}
```

Comment:
I worry that this logic may not quite work.  The issue I _think_ I am seeing is that an hvac system can server more than one zone (the schema seems to allow that at least).  That means that the same `zone_id_b` can occur in more than one zone_list.  Any `zone_info[zone_id_b]` after the first one will overwrite what was already there.

I _think_ the solution is to put `dict_of_zones_and_terminal_units_served_by_hvac_sys_b` in `data` and then put the rest of the logic in `calc_vals` method.

Wherever a function here uses `rmi_b` you'll need to create a dictionary holding all the calls for that function.  For example:
```
are_all_hvac_sys_fan_objs_autosized_dict = {
    hvac_id_b: are_all_hvac_sys_fan_objs_autosized(rmi_b, hvac_id_b) for
    hvac_id_b in dict_of_zones_and_terminal_units_served_by_hvac_sys_b
}
```

Reply:
Great point! The fundamental issue was the ```zone_info``` dict had the ```zone_id```. Now, it was changed to have ```hvac_id``` instead. Thanks for this cooment!